roxctl/netpol/connectivity/map/map.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 const (
-	defaultOutputFileNamePrefix = "connlist."
-	defaultOutputFormat         = "txt"
+	defaultOutputFileNamePrefix  = "connlist."
+	exposureOutputFileNamePrefix = "exposure."
```

Comment:
why is this required? it's the same connlist command, so I would not change the name of the default file name based on some arg...

Reply:
i see.
i did this for case we want to run on same dir twice (with and without the `exposure` flag, thought maybe not to "erase" previous results), however a user may choose to provide his output file name to avoid this